kind/kind.sh

```diff
@@ -151,11 +151,84 @@ if [ "$VERBOSE" == true ]; then
     set -o xtrace
 fi
 
+function wait_for_pods {
+  for ns in "$namespace"; do
+    for pod in $(kubectl get pods -n $ns | grep -v NAME | awk '{ print $1 }'); do
+      counter=0
+      echo kubectl get pod $pod -n $ns
+      kubectl get pod $pod -n $ns
+      while [[ $(kubectl get pods $pod -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}' -n $ns) != True ]]; do
+        sleep 1
+        let counter=counter+1
+
+        if ((counter == $sleep)); then
+          echo "POD $pod failed to start in $sleep seconds"
+            kubectl get events -n $ns --sort-by='.lastTimestamp'
+          echo "Exiting"
+
+          exit -1
+        fi
+      done
+    done
+  done
+}
+
+# Install Calico in Controller...
+echo Switch to controller context and Install Calico...
+kubectx $PREFIX$CONTROLLER
+kubectx
+
+echo Install the Tigera Calico operator...
+kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
+
+echo Install the custom resource definitions manifest...
+kubectl create -f https://projectcalico.docs.tigera.io/manifests/custom-resources.yaml
+sleep 120
+
+echo "Check for Calico namespaces, pods"
+kubectl get ns
+kubectl get pods -n calico-system
+echo "Wait for Calico to be Running"
+namespace=calico-system
+sleep=900
```

Comment:
Sleep 900 seems excessive?   If it's not going to work, could we fail faster?   Fwiw, in my testing it tool 32sec for calico to be ready... so 60 or 90 might be fine?